README.md

```diff
@@ -57,15 +57,15 @@ Please refer to the [Getting Started Guide](https://docs.spring.io/spring-ai/ref
 ## Cloning the repo
 
 This repository contains [large model files](https://github.com/spring-projects/spring-ai/tree/main/models/spring-ai-transformers/src/main/resources/onnx/all-MiniLM-L6-v2).
-To clone it you have to either:
+To clone it you have to:
 
-- Ignore the large files (won't affect the spring-ai behaviour) :  `GIT_LFS_SKIP_SMUDGE=1 git clone git@github.com:spring-projects/spring-ai.git`.
-- Or install the [Git Large File Storage](https://git-lfs.com/) before cloning the repo.
+- Install the [Git Large File Storage](https://git-lfs.com/) before cloning the repo.
```

Comment:
I frankly don't even know this is needed.  I never had an issue cloning (on ubuntu) and I others on the team on Mac don't have this large file support installed.  Can you tell us what happens?  I'd like to not have this dependency to clone of course ,but there seems to be a deeper issue here.  

Thanks!

Reply:
I don't have git-lfs installed either (macos) and never saw an error message